controllers/api.js

```diff
@@ -1,33 +1,40 @@
-var _ = require('lodash');
-var async = require('async');
+const _ = require('lodash');
+const async = require('async');
 
 /**
  * Split into declaration and initialization for better startup performance.
  */
-var validator;
-var cheerio;
-var graph;
-var LastFmNode;
-var tumblr;
-var foursquare;
-var Github;
-var Twit;
-var stripe;
-var twilio;
-var Linkedin;
-var BitGo;
-var clockwork;
-var paypal;
-var lob;
-var ig;
-var Y;
-var request;
+const validator = require('validator');
```

Comment:
Yeah keep it as is for now. Maybe you can just change `var` to `let`. I will do some more performance testing later to determine the best approach.